### ISSUE TITLE
flutter build web error

### ISSUE THREAD

**Issue by sleepreading** [leanflutter/launch_at_startup#2]
```
Error: Only JS interop members may be 'external'.
  external int wBitsPerSample;
....
....
/D:/tmp/.pub/cache/hosted/pub.flutter-io.cn/win32-2.3.1/lib/src/combase.dart:24:37:
Error: Only JS interop members may be 'external'.
  external Pointer<Pointer<IntPtr>> lpVtbl;
                                    ^
Error: Compilation failed.
```

**Comment by sleepreading** [comment]
import launch_at_startup, but not used

```
Future launchStartup() async {
    if (!isDesktop) {
      return;
    }
    // launch at startup
    LaunchAtStartup.instance.setup(
      appName: packageInfo.appName,
      appPath: Platform.resolvedExecutable, // import 'dart:io';
    );

    isLaunchAtStartup = await LaunchAtStartup.instance.isEnabled();
  }
```

**Comment by lijy91** [comment]
Fixed, please upgrade to `v0.1.2`
